### PR TITLE
limit PR tests to 3.9/4.11, newer releases can use GHA

### PR DESCRIPTION
**Pull request by evgeni**
needs
- https://github.com/theforeman/foreman/pull/10132
- https://github.com/Katello/katello/pull/10968

### CODE REVIEW

**Comment by ekohl** [comment]
Don't we want to pick to 3.9/4.11 too so it's consistent?

**Comment by evgeni** [comment]
we surely can. I was playing safe here.

**Comment by ekohl** [comment]
Let's play it less safe: https://github.com/theforeman/foreman/pull/10134 and then drop them all.

**Comment by ekohl** [comment]
And https://github.com/Katello/katello/pull/10974.

File: branch-foreman [hunk 1]
```diff
@@ -32,26 +32,19 @@ echo "Create theforeman.org/yaml/jobs/test_${FOREMAN_VERSION/./_}_stable.yaml"
 echo "Add ${FOREMAN_VERSION%*.} as a minor to theforeman.org/yaml/views/release.yml"
 
 # TODO The sed line was too complex
-echo "Add the following to the foreman_integration_versions section in theforeman.org/pipelines/test/foreman/vars.groovy"
+echo "Update .github/workflows/foreman.yml in foreman.git and set plugin_version for Katello like this:"
```
Comment (ekohl) [inline branch-foreman:35]: I think this branch script should be limited to just this repository.

**Comment by ekohl** [comment]
Alternative was merged.

**Pull request closed without merging**